examples/platforms/cc2538/radio.c

```diff
@@ -103,6 +103,11 @@ void enableReceiver(void)
 
 void disableReceiver(void)
 {
+    if (!sIsReceiverEnabled)
```

Comment:
Is this necessary?  If it is necessary, would prefer to either use `VerifyOrExit` or put the code below within the if-statement as is done in `enableReceiver`.  I'd like to maintain the code style of having only a single exit point for each method/function.